### PR TITLE
Fix: network reuse

### PR DESCRIPTION
**Pull request by papey**
Reuse existing network if exists

### CODE REVIEW

**Comment by athoune** [comment]
subnet ending with 0.0 is weird. Last 0 is ok, it's a /24, but IP[2] should not be a 0 all the time.

File: compose/network.go [hunk 1]
```diff
@@ -79,9 +96,6 @@ func (n *Networks) New(project string) (string, error) {
 	for i := 0; i < 2; i++ {
 		now = time.Now()
 		subnet, err := _network.NextAvailableNetwork(subnets, n.min, n.max, n.mask)
```
Comment (athoune) [inline compose/network.go:98]: what happens if `err != nil` ? subnet is unusable.

**Review by athoune — CHANGES_REQUESTED** [review]
How to handle NextAvailableNetwork ?

File: compose/network.go (hunk 1 above)
Comment (papey) [reply compose/network.go]: Err nil check added

**Review by athoune — APPROVED** [review]

**Pull request merged**